roles/ceph-common/tasks/checks/check_system.yml

```diff
@@ -26,8 +26,3 @@
   changed_when: false
   sudo: false
   register: ansible_version
-
-- name: fail on unsupported ansible version
-  fail:
-    msg: "Ansible version must be >= 1.9, please update!"
-  when: "{{ ansible_version.stdout | version_compare('1.9', '<') }}"
```

Comment:
@erikh 

just wondering, I think if we get rid of the curly braces this might work. Do you really want to remove the task itself?